backend/tests/v5/helper/services.js

```diff
@@ -56,19 +56,31 @@ queue.purgeQueues = async () => {
 	}
 };
 
+const defaultTestRoleName = 'userTest';
+
 // userCredentials should be the same format as the return value of generateUserCredentials
 db.createUser = async (userCredentials, tsList = [], customData = {}) => {
 	const { user, password, apiKey, basicData = {} } = userCredentials;
 	const roles = tsList.map((ts) => ({ db: ts, role: 'team_member' }));
+	roles.push({ db: 'admin', role: defaultTestRoleName });
 	const adminDB = await DbHandler.getAuthDB();
 	return adminDB.addUser(user, password, { customData: { ...basicData, ...customData, apiKey }, roles });
 };
 
 db.createTeamspaceRole = (ts) => createTeamSpaceRole(ts);
 
+db.createDefaultRole = async () => {
+	const roleFound = await DbHandler.findOne('admin', 'system.roles', { _id: `admin.${defaultTestRoleName}` });
+	if (!roleFound) {
+		const createRoleCmd = { createRole: defaultTestRoleName, privileges: [], roles: [] };
+		await DbHandler.runCommand('admin', createRoleCmd);
+	}
+};
+
 // breaking = create a broken schema for teamspace to trigger errors for testing
-db.createTeamspace = (teamspace, admins = [], breaking = false) => {
+db.createTeamspace = async (teamspace, admins = [], breaking = false) => {
 	const permissions = admins.map((adminUser) => ({ user: adminUser, permissions: TEAMSPACE_ADMIN }));
+	await db.createDefaultRole();
```

Comment:
and this

Reply:
I added these to get rid of these warnings

![image](https://user-images.githubusercontent.com/71635617/166917439-65604c5f-c6aa-4ac0-b85a-10f448776825.png)

Reply:
you shouldn't need to do this with the change where we create the defaultRole before creating the user